### PR TITLE
Anpassung der external worker

### PR DESCRIPTION
**Pull request by EnnoKm**
- Erstellung / Einführung des von camunda bereitgestellten ExternalWorker in den calculator-service
- Löschen der alten FetchExternalTaskJobProcessor & ExecuteExternalTaskJobProcessor im .route package
- Löschen der nicht mehr benötigten externalTaskRoute in der CalculateRouteBuilder Klasse

### CODE REVIEW

File: calculator-service/src/main/resources/application.yaml [hunk 1]
```diff
@@ -1,3 +1,6 @@
 server.port: 8090
 camunda.rest.baseurl: http://localhost:8080/engine-rest/
 camunda.externalTask.workerId: apacheCamelWorker
```
Comment (mschulte) [inline calculator-service/src/main/resources/application.yaml:3]: that workerId makes no sense anymore does it? :)

**Review by mschulte — COMMENTED** [review]
please do not commit binaries (camunda-databases) here. we should add them to .gitignore

File: calculator-service/src/main/java/de/viadee/camunda/connected/calculator/externaltaskworker/ExternalTaskCalculatorHandler.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	@Value("${lockDuration}")
+	private int lockduration;
+	
+	@Value("${topicSubscription}")
+	private String topicSubscription;
+
+	@Override
+	public void execute(ExternalTask externalTask, ExternalTaskService externalTaskService) {
+
+		long num1x = (long) externalTask.getVariable("num1");
```
Comment (mschulte) [inline calculator-service/src/main/java/de/viadee/camunda/connected/calculator/externaltaskworker/ExternalTaskCalculatorHandler.java:40]: better we use getVariableTyped from the camunda-API?

**Comment by mschulte** [comment]
i added one further comment. If we got that fixed we are ready to merge ;)

File: calculator-service/src/main/java/de/viadee/camunda/connected/calculator/externaltaskworker/ExternalTaskCalculatorHandler.java (hunk 2 above)
Comment (rnschk) [reply calculator-service/src/main/java/de/viadee/camunda/connected/calculator/externaltaskworker/ExternalTaskCalculatorHandler.java]: why not simply use `long num1x = externalTask.getVariable("num1");` without casting, as the method is already generic?

**Comment by mschulte** [comment]
Looks good for me now. lets get this merged :)

**Pull request merged**